Resources/Prototypes/_Sunrise/Entities/Objects/Clothing/Pouches/pouches.yml

```
@@ -0,0 +1,87 @@
+- type: entity
+  parent: [BaseClothingBeltSounds, ClothingBeltStorageBase]
+  id: Pouch
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)